src/stlink-lib/flash_loader.h

```diff
@@ -12,16 +12,10 @@
 
 #include <stlink.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif
 
 int stlink_flash_loader_init(stlink_t *sl, flash_loader_t* fl);
 int stlink_flash_loader_write_to_sram(stlink_t *sl, stm32_addr_t* addr, size_t* size);
 int stlink_flash_loader_run(stlink_t *sl, flash_loader_t* fl, stm32_addr_t target, const uint8_t* buf, size_t size);
 
-#ifdef __cplusplus
-}
-#endif
 
```

Comment:
```suggestion
```

Reply:
Addressed in commit da3d9e3f1e1a6d116df5830cf506f30ff56196b3.